### ISSUE TITLE
Add missing a11y ids to TrackingProtectionViewController

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#25330]
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-11629)
